nano/node/distributed_work.cpp

```diff
@@ -257,6 +258,11 @@ void nano::distributed_work::set_once (boost::optional<uint64_t> work_a)
 	if (!completed.exchange (true))
 	{
 		callback (work_a);
+		if (node.config.logging.work_generation_time ())
+		{
+			node.logger.try_log (boost::str (boost::format ("Work generation for %1%, with a threshold difficulty of %2% (multiplier %3%x) complete: %4% ms")
+			% root.to_string () % nano::to_string_hex (difficulty) % nano::to_string (nano::difficulty::to_multiplier (difficulty, node.network_params.network.publish_threshold), 2) % elapsed.stop ().count ()));
```

Comment:
This statement is really long, can `"Work generation for %1%, with a threshold difficulty of %2% (multiplier %3%x) complete: %4% ms"` and `nano::to_string (nano::difficulty::to_multiplier (difficulty, node.network_params.network.publish_threshold), 2)` be put into local variables? that should shorten it a lot.

Reply:
Done, makes sense!